src/test/test-calendarspec.c

```diff
@@ -47,8 +47,11 @@ static void _test_next(int line, const char *input, const char *new_tz, usec_t a
         if (old_tz)
                 old_tz = strdupa_safe(old_tz);
 
-        if (!isempty(new_tz))
-                new_tz = strjoina(":", new_tz);
+        if (!isempty(new_tz)) {
+                if (strchr(new_tz, ',') == NULL) {
+                        new_tz = strjoina(":", new_tz);
+                }
+        }
```

Comment:
please follow coding style: no {} around single line if blocks. And we do not spell out `== NULL` for pointer checks